ckanext/opendata_theme/extended_themes/bostonma/templates/page.html

```diff
@@ -5,15 +5,7 @@
 <div class="main-navigation">
   <div class="main-navigation-title"></div>
     <div class="region region-navigation">
-      <ul class="menu list-unstyled unstyled">
-      <li class="menu__item is-leaf first leaf secondary-menu-item">
-        <a href="{{ h.url_for('/') }}" class="menu__link">{{ _('Home') }}</a>
-      </li>
-      <li class="menu__item is-leaf first leaf secondary-menu-item">
-        <a href="https://www.boston.gov/" class="menu__link" target="_blank">{{ _('Boston.gov') }}</a>
-      </li>
-
-      <div class="side-nav-account">
+      <div class="side-nav-account"></div>
```

Comment:
Did you mean to close the div for `side-nav-account` here? In this case the extra closing div should be removed so that line 8 is just `<div class="side-nav-account">`

Otherwise it look like the following:
![Screenshot 2024-10-02 at 12 37 28 PM](https://github.com/user-attachments/assets/15ef4867-3634-448f-a891-8178660b8e5b)